core/src/main/java/org/apache/iceberg/BaseScan.java

```diff
@@ -59,8 +59,8 @@ protected TableScanContext context() {
     return context;
   }
 
-  protected abstract T newRefinedScan(
-      TableOperations newOps, Table newTable, Schema newSchema, TableScanContext newContext);
+  protected abstract T newRefinedScan(TableOperations newOps, Table newTable, Schema newSchema,
+                                      TableScanContext newContext);
```

Comment:
I don't think these reformatting changes are worth it, since the alternative is already in master.

Reply:
ok. will revert